sources/packages/backend/apps/api/src/route-controllers/student-scholastic-standings/_tests_/e2e/student-scholastic-standings.institutions.controller.saveScholasticStanding.e2e-spec.ts

```diff
@@ -0,0 +1,321 @@
+      },
+    };
+    // Mock the form service to validate the dry-run submission result.
+    // TODO: Form service must be hosted for E2E tests to validate dry run submission
+    // and this mock must be removed.
+    const formService = await getProviderInstanceForModule(
+      appModule,
+      AppInstitutionsModule,
+      FormService,
+    );
+    const dryRunSubmissionMock = jest.fn().mockResolvedValue({
+      valid: true,
+      formName: SCHOLASTIC_STANDING_FORM_NAME,
+      data: { data: payload.data },
+    });
+    formService.dryRunSubmission = dryRunSubmissionMock;
+    const endpoint = `/institutions/scholastic-standing/location/${collegeFLocation.id}/application/0000`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.UNPROCESSABLE_ENTITY)
+      .expect({
+        message:
+          "Application Not found or invalid current assessment or offering.",
+        errorType: APPLICATION_NOT_FOUND,
+      });
+  });
+
+  it("Should throw unprocessable entity exception error when the application is not found.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(db.dataSource, {
+      institutionLocation: collegeFLocation,
+    });
+    application.isArchived = true;
+    await db.application.save(application);
+    const payload: ScholasticStandingAPIInDTO = {
+      data: {
+        booksAndSupplies: 1000,
+        dateOfChange: getISODateOnlyString(new Date()),
+        scholasticStandingChangeType:
+          StudentScholasticStandingChangeType.ChangeInIntensity,
+      },
+    };
+    // Mock the form service to validate the dry-run submission result.
+    // TODO: Form service must be hosted for E2E tests to validate dry run submission
+    // and this mock must be removed.
+    const formService = await getProviderInstanceForModule(
+      appModule,
+      AppInstitutionsModule,
+      FormService,
+    );
+    const dryRunSubmissionMock = jest.fn().mockResolvedValue({
+      valid: true,
+      formName: SCHOLASTIC_STANDING_FORM_NAME,
+      data: { data: payload.data },
+    });
+    formService.dryRunSubmission = dryRunSubmissionMock;
+    const endpoint = `/institutions/scholastic-standing/location/${collegeFLocation.id}/application/${application.id}`;
+
+    // Act/Assert
+    await request(app.getHttpServer())
+      .post(endpoint)
+      .send(payload)
+      .auth(institutionUserToken, BEARER_AUTH_TYPE)
+      .expect(HttpStatus.UNPROCESSABLE_ENTITY)
+      .expect({
+        message: "This application is no longer eligible to request changes.",
+        errorType: APPLICATION_CHANGE_NOT_ELIGIBLE,
+      });
+  });
+
+  it("Should throw unprocessable entity exception error when the application status is not complete.", async () => {
+    // Arrange
+    const application = await saveFakeApplicationDisbursements(db.dataSource, {
```

Comment:
Do we need to have disbursements also saved to allow the test or `saveFakeApplication` would be enough?